client/src/main/java/org/spine3/base/Stringifiers.java

```diff
@@ -110,7 +116,7 @@ protected Timestamp doBackward(String s) {
             try {
                 return Timestamps.parse(s);
             } catch (ParseException ignored) {
-                throw conversionArgumentException("Occurred exception during conversion");
+                throw conversionArgumentException(CONVERSION_EXCEPTION);
```

Comment:
That branch is tested with the  `throw_exception_when_string_cannot_be_converted_to_timestamp` test method.